### ISSUE TITLE
Consider revealing cells radially outwards

### ISSUE THREAD

**Issue by JayaSuryaT** [JayaSuryaT/minesweeper-j-compose#33]
Currently, when the first cell is revealed (or whenever a bunch of cells gets revealed due to any action), the _cell revelation_ order is like:
> "Diagonally reveal up to the topmost (and the leftmost) cell (which is getting revealed), and then start revealing linearly one by one from left-to-right then top-to-bottom."

This works fine and looks good enough when the game is totally zoomed out, as it is easier to follow. 

But, when the player is zoomed in a fair bit, and if he reveals a cell, which in turn reveals a bunch of other cells, and cells offscreen start getting revealed. **Here comes the problem**, based on the position of cells and state of the board, the adjacent cells right next to the revealed cell might not get revealed in the first sweep and may take some to get revealed.

This particular problem is partially being solved by providing `aural and haptic feedback`. With this, the player should (ideally) get to know that _cell revelation_ is in progress. Again, this is not a perfect solution to this problem, but it does help a bit.

One more thing that could help curb this problem is changing the revelation order to radially outwards from the clicked cell.

## TLDR : Reveal cells radially outwards from the clicked cell.
